SQLhomework 2.sql

```diff
@@ -0,0 +1,165 @@
+CREATE DATABASE LibrariesManagement;
+
+USE LibrariesManagement;
+
+CREATE TABLE Libraries
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    Name VARCHAR(50),
+    StreetName VARCHAR(50),
+    City VARCHAR(50)
+);
+CREATE TABLE Librarians
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    Name VARCHAR(30),
+    HireDate Date,
+    IsOnHoliday VARCHAR(3),
+    LibraryId INT REFERENCES Libraries(Id)
+);
+
+CREATE TABLE Books
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    Title VARCHAR(50),
+    AuthorFirstName VARCHAR(30),
+    AuthorLastName VARCHAR(30),
+    Type VARCHAR(10), -- crime, fantasy, horror, adventure
+);
+
+CREATE TABLE Books_Libraries
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    BookId INT REFERENCES Books(Id),
+    LibraryId INT REFERENCES Libraries(Id)
+);
+
+CREATE TABLE Sales
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    NumberOfCopies INT,
+    LibraryId INT REFERENCES Libraries(Id),
+    LibrarianId INT REFERENCES Librarians(Id)
+);
+
+CREATE TABLE Books_Sales
+(
+    Id INT IDENTITY(1, 1) PRIMARY KEY,
+    BookId INT REFERENCES Books(Id),
+    SalesId INT REFERENCES Sales(Id)
+);
+
+INSERT INTO Libraries(Name,StreetName,City)
+VALUES('Antia','Vointei','Bacau'),('Cartes','Piperului','Sibiu'),('Veselie','Olarilor','Brasov'),('Tacuta','Fericirii','Iasi'),('Elevului','Verde','Bacau'),('Noua','Palat','Iasi');
```

Comment:
It would help if all these values would be added one under the other. It still works and it's easier to read. Example:
```
INSERT INTO Libraries(Name,StreetName,City)
VALUES
('Antia','Vointei','Bacau'),
('Cartes','Piperului','Sibiu')
```